[2024-06-14T10:46:53.227+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-14T10:46:53.248+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: NYC_Taxi_pipeline.data_delivery manual__2024-06-14T10:45:14.563867+00:00 [queued]>
[2024-06-14T10:46:53.256+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: NYC_Taxi_pipeline.data_delivery manual__2024-06-14T10:45:14.563867+00:00 [queued]>
[2024-06-14T10:46:53.256+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-14T10:46:53.268+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): data_delivery> on 2024-06-14 10:45:14.563867+00:00
[2024-06-14T10:46:53.275+0000] {standard_task_runner.py:63} INFO - Started process 113 to run task
[2024-06-14T10:46:53.277+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'NYC_Taxi_pipeline', 'data_delivery', 'manual__2024-06-14T10:45:14.563867+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/NYC_Taxi_pipeline.py', '--cfg-path', '/tmp/tmpr000yf8c']
[2024-06-14T10:46:53.281+0000] {standard_task_runner.py:91} INFO - Job 25: Subtask data_delivery
[2024-06-14T10:46:53.322+0000] {task_command.py:426} INFO - Running <TaskInstance: NYC_Taxi_pipeline.data_delivery manual__2024-06-14T10:45:14.563867+00:00 [running]> on host 762c91fd0326
[2024-06-14T10:46:53.590+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Mahmoud' AIRFLOW_CTX_DAG_ID='NYC_Taxi_pipeline' AIRFLOW_CTX_TASK_ID='data_delivery' AIRFLOW_CTX_EXECUTION_DATE='2024-06-14T10:45:14.563867+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-14T10:45:14.563867+00:00'
[2024-06-14T10:46:53.590+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-14T10:46:53.601+0000] {NYC_Taxi_pipeline.py:54} INFO - data_delivery started
[2024-06-14T10:46:53.601+0000] {Load_Data.py:19} INFO - loading location Dimension ...
[2024-06-14T10:46:53.605+0000] {Table_creation.py:30} INFO - Loading data into table DimLocation...
[2024-06-14T10:46:53.607+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-14T10:46:53.658+0000] {Table_creation.py:45} INFO - Data loaded successfully into table DimLocation With 265 Row
[2024-06-14T10:46:53.659+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-14T10:46:53.659+0000] {Load_Data.py:22} INFO - DimLocation_df dimension loaded successfully
[2024-06-14T10:46:53.659+0000] {Load_Data.py:28} INFO - loading DimRate Dimension ...
[2024-06-14T10:46:53.661+0000] {Table_creation.py:30} INFO - Loading data into table DimRate...
[2024-06-14T10:46:53.664+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-14T10:46:53.669+0000] {Table_creation.py:45} INFO - Data loaded successfully into table DimRate With 6 Row
[2024-06-14T10:46:53.669+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-14T10:46:53.669+0000] {Load_Data.py:31} INFO - DimRate_df dimension loaded successfully
[2024-06-14T10:46:53.670+0000] {Load_Data.py:46} INFO - loading payment dimension....
[2024-06-14T10:46:53.671+0000] {Table_creation.py:30} INFO - Loading data into table DimPayment...
[2024-06-14T10:46:53.674+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-14T10:46:53.679+0000] {Table_creation.py:45} INFO - Data loaded successfully into table DimPayment With 6 Row
[2024-06-14T10:46:53.680+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-14T10:46:53.680+0000] {Load_Data.py:49} INFO - payment_df dimension loaded successfully
[2024-06-14T10:46:53.680+0000] {Load_Data.py:10} INFO - loading vendor Dimension ...
[2024-06-14T10:46:53.682+0000] {Table_creation.py:30} INFO - Loading data into table DimVendor...
[2024-06-14T10:46:53.684+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-14T10:46:53.689+0000] {Table_creation.py:45} INFO - Data loaded successfully into table DimVendor With 2 Row
[2024-06-14T10:46:53.689+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-14T10:46:53.689+0000] {Load_Data.py:13} INFO - vendor_df dimension loaded successfully
[2024-06-14T10:46:53.689+0000] {Load_Data.py:37} INFO - loading trip_type dimension....
[2024-06-14T10:46:53.691+0000] {Table_creation.py:30} INFO - Loading data into table DimTrip_type...
[2024-06-14T10:46:53.694+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-14T10:46:53.701+0000] {Table_creation.py:45} INFO - Data loaded successfully into table DimTrip_type With 2 Row
[2024-06-14T10:46:53.701+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-14T10:46:53.701+0000] {Load_Data.py:40} INFO - trip_df dimension loaded successfully
[2024-06-14T10:46:53.701+0000] {Load_Data.py:55} INFO - loading fact table ....
[2024-06-14T10:47:03.369+0000] {local_task_job_runner.py:310} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2024-06-14T10:47:03.370+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-14T10:47:03.371+0000] {process_utils.py:132} INFO - Sending 15 to group 113. PIDs of all processes in the group: [113]
[2024-06-14T10:47:03.372+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 113
[2024-06-14T10:47:03.372+0000] {taskinstance.py:2611} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-06-14T10:47:03.439+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-14T10:47:03.504+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=113, status='terminated', exitcode=0, started='10:46:52') (113) terminated with exit code 0
